### ISSUE TITLE
Autohide_toolbox.css breaks the "Focus Address Bar" keyboard shortcut

### ISSUE THREAD

**Issue by RJKAJoaquin** [MrOtherGuy/firefox-csshacks#440]
I have Autohide_toolbox.css enabled right now and when I press Ctrl + L (the keyboard shortcut to focus the address bar), it either doesn't pull up the url bar at all or it makes it appear but doesn't focus on it. This means that, while the address bar is revealed, I still have to click on the url bar to type on it. The normal behaviour is that I'd be able to type on the address bar immediately after pressing Ctrl + L.

**Comment by MrOtherGuy** [comment]
Yeah, the quick and dirty workaround I had added which made urlbar have visibility:hidden also prevented it from receiving focus. I've now added a better solution for hiding it which also fixes couple of other rather annoying problems that it had - hopefully without introducing new ones.
